File: js/day37.js
const now = new Date();

// Set 'tonight' to tomorrow at 00:00:00:000
const tonight = new Date(now.getFullYear(), now.getMonth(), now.getDate() + 1);

// Difference in milliseconds
const remainingMs = tonight - now;

// Convert milliseconds to microseconds
const microSeconds = remainingMs * 1000;
console.log(`Remaining Microseconds Today: ${microSeconds}`);W

File: python/day37.py
from datetime import datetime, time

now = datetime.now();

# Define the end of the day (23:59:59.999999)
end_of_day = datetime.combine(now.date(), time.max)

# Calculate the time remaining
remaining = end_of_day - now

# Convert total remaining time to microseconds
remaining_micros = int(remaining.total_seconds() * 1_000_000)

print(f"Remaining Microseconds Today: {remaining_micros}")